src/http/request.cr

```diff
@@ -285,7 +285,7 @@ class HTTP::Request
   end
 
   # Sets request's query component.
-  def query=(value)
+  def query=(value : String) : String
```

Comment:
This one caused a regression in Athena since before you _could_ pass `nil` as per https://crystal-lang.org/api/URI.html#query%3D%28query%3AString%7CNil%29-instance-method